hi-ml-multimodal/requirements_build.txt

```diff
@@ -1,3 +1,4 @@
+colorama==0.4.4
```

Comment:
what's this about?

Reply:
I have the same question - to add some context, hi-ml-multimodal does not use any of the other packages in this repo, so it should be unaffected by this PR.